src/services/logger.js

```diff
@@ -0,0 +1,30 @@
+const { createLogger, format, transports } = require('winston');
+
+const { combine, timestamp, prettyPrint } = format;
+
+/**
+ *  
+ *  levels: {
+    error: 0,
+    warn: 1,
+    info: 2
+    }
+ * 
+ * 
+ */
+const logger = createLogger({
+  level: 'info',
+  format: combine(timestamp(), prettyPrint()),
+  transports: [
+    new transports.File({ filename: 'error.log', level: 'error' }), // error
+    new transports.File({ filename: 'combined.log' }), // combined file for info and below
```

Comment:
Change the comments next to both these lines as:
`log file for error logs`
`combined log file for info and verbose logs`

Reply:
Also can we have both the log files inside a folder called `logs`?